cmd/genutils/genutils.go

```diff
@@ -41,3 +43,17 @@ func OutDir(path string) (string, error) {
 	outDir = outDir + "/"
 	return outDir, nil
 }
+
+// ParseKubeConfigFiles gets a string that contains one or multiple kubeconfig files.
+// If there are more than one file, separated by comma
```

Comment:
should the stringSliceVar from command line args flag take care of this parsing already ?

Reply:
As I explained earlier, I prefer let internal logic decide treat the argument as a string or array. This logic applies to both single or multiple kubeconfigs without option parsing changes.